node/src/Commands.ts

```diff
@@ -1850,6 +1850,41 @@ export function createGeoAdd(
     return createCommand(RequestType.GeoAdd, args);
 }
 
+/**
+ * Enumeration representing distance units options for the {@link geodist} command.
+ */
+export enum GeoUnit {
```

Comment:
Should we store it in `src/commands`?

Reply:
I thought we are keeping it in commands for now